syntax = "proto2";
package mmo.proto;

import "loot_entry.proto";

message ItemLoot {
	repeated LootEntry entry = 1;
}
